packages/circle-demo-webapp/app/tailwind.css

```diff
@@ -3,95 +3,143 @@
-    --border: 240 3.7% 15.9%;
-    --input: 240 3.7% 15.9%;
-    --ring: 240 4.9% 83.9%;
-
-    /* Chart Colors */
-    --chart-1: 220 70% 50%;
-    --chart-2: 160 60% 45%;
-    --chart-3: 30 80% 55%;
-    --chart-4: 280 65% 60%;
-    --chart-5: 340 75% 55%;
-  }
+    :root {
+        /* Background Colors */
+        --background: 0 0% 100%; /* White background for light mode */
+        --foreground: 240 10% 3.9%; /* Dark text for light mode */
+
+        /* Card and Popover Colors */
+        --card: 0 0% 100%;
+        --card-foreground: 240 10% 3.9%;
+        --popover: 0 0% 100%;
+        --popover-foreground: 240 10% 3.9%;
+
+        /* Primary Brand Color - Jelly 400 */
+        --primary: 255 82% 64%; /* Jelly 400 - Vibrant purple */
+        --primary-foreground: 0 0% 98%; /* White text for contrast */
+
+        /* Secondary and Accent Colors */
+        --secondary: 240 4.8% 95.9%;
+        --secondary-foreground: 240 5.9% 10%;
+        --muted: 240 4.8% 95.9%;
+        --muted-foreground: 240 3.8% 46.1%;
+        --accent: 240 4.8% 95.9%;
+        --accent-foreground: 240 5.9% 10%;
+
+        /* Destructive and Error Colors */
+        --destructive: 0 84.2% 60.2%;
+        --destructive-foreground: 0 0% 98%;
+
+        /* Utility Colors */
+        --border: 240 5.9% 90%;
+        --input: 240 5.9% 90%;
+        --ring: 240 10% 3.9%;
+
+        /* Chart Colors */
+        --chart-1: 12 76% 61%;
+        --chart-2: 173 58% 39%;
+        --chart-3: 197 37% 24%;
+        --chart-4: 43 74% 66%;
+        --chart-5: 27 87% 67%;
+
+        /* Radius for rounded corners */
+        --radius: 0.5rem;
+    }
+
+    .dark {
+        /* Background Colors */
+        --background: 240 10% 3.9%; /* Dark background for dark mode */
+        --foreground: 0 0% 98%;
+
+        /* Card and Popover Colors */
+        --card: 240 10% 3.9%;
+        --card-foreground: 0 0% 98%;
+        --popover: 240 10% 3.9%;
+        --popover-foreground: 0 0% 98%;
+
+        /* Primary Brand Color - Jelly 400 */
+        --primary: 255 82% 64%; /* Jelly 400 - Vibrant purple */
+        --primary-foreground: 0 0% 98%; /* White text for contrast */
+
+        /* Secondary and Accent Colors */
+        --secondary: 240 3.7% 15.9%;
+        --secondary-foreground: 0 0% 98%;
+        --muted: 240 3.7% 15.9%;
+        --muted-foreground: 240 5% 64.9%;
+        --accent: 240 3.7% 15.9%;
+        --accent-foreground: 0 0% 98%;
+
+        /* Destructive and Error Colors */
+        --destructive: 0 62.8% 30.6%;
+        --destructive-foreground: 0 0% 98%;
+
+        /* Utility Colors */
+        --border: 240 3.7% 15.9%;
+        --input: 240 3.7% 15.9%;
+        --ring: 240 4.9% 83.9%;
+
+        /* Chart Colors */
+        --chart-1: 220 70% 50%;
+        --chart-2: 160 60% 45%;
+        --chart-3: 30 80% 55%;
+        --chart-4: 280 65% 60%;
+        --chart-5: 340 75% 55%;
+
+        .dialog-title {
+            color: #ffffff;
+        }
+    }
+
+    .dialog-base {
```

Comment:
Honestly, I don’t think this is the right approach for overriding the default theme styles. First, we should avoid using any custom class names. I’m confident we can handle everything using Tailwind’s built-in utility classes. For example, if the dialog title needs a font size adjustment, it should be updated directly in the dialog component within the ui folder.